CHANGELOG.rst

```diff
@@ -8,6 +8,13 @@ TBR
 * Provider for ``web_poet.ResponseUrl`` is added, which allows to access the
   response URL in the page object. This triggers a download unlike the provider
   for ``web_poet.RequestUrl``.
+* Move from web-poet 0.5.0 to 0.6.0.
```

Comment:
I think we should say here that web-poet 0.6+ is now required. We're not moving to 0.6.0, because web-poet versions > 0.6.0 should be supported as well.

Reply:
Nice catch! Will do this alongside updating `setup.py` and `tox.ini` later on.